mkdocs.yml

```diff
@@ -116,6 +116,7 @@ plugins:
             - css/watermark.css
   - macros:
       module_name: tools/mkdocs_macros_bids/main
+      on_error_fail: true
```

Comment:
adding this extra check because the wesbite would still build but with broken pages which is not something we want